app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = t "layouts.application.title"
+    page_title.empty? ? base_title : page_title + " | " + base_title
```

Comment:
so sánh nil?, empty?, blank?, present? 
nên dùng cái nào trong trường hợp này

Reply:
- nil là dùng để check nil (null), nil có thể sử dụng với bất kì object nào
- empty dùng để check xem object có rỗng ko, nó sử dụng được với array, hash, string
- blank dùng cho bất kỳ object nào, trả về true nếu object là giá trị là nil, false, empty
- present dùng để check object có NOT nil, NOT false, hay là NOT empty 

-> nên dùng blank trong TH này